test/apiClients/AirtableApiClient.ts

```diff
@@ -1,8 +1,14 @@
 import AirTableApiClient, { AirTableFilters, Organization } from '../../src/apiClients/AirTableApiClient';
+
+import { ErrorObject } from '../../src/apiClients/AirTableApiClient';
 import env from '../../src/env';
 import nock from 'nock';
+import sinon from 'sinon';
 import test from 'ava';
 
+// eslint-disable-next-line
+const objectToQueryParamString = require('airtable/lib/object_to_query_param_string.js');
```

Comment:
lol i do remember we discussed this, but just add a quick comment as to why you disabled the lint here

Reply:
I am using the function from the AirTable code base that they use to convert objects to QueryStrings for post requests (to use with nock). It doesn't get recognized as an import since its type is any so I used a require to be able to use the function.